com.haysell.pos.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id":"com.haysell.pos",
+    "runtime":"org.kde.Platform",
+    "runtime-version":"5.15-23.08",
+    "sdk":"org.kde.Sdk",
+    "command":"haysell-pos",
+    "finish-args":[
+      "--share=ipc",
+      "--share=network",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=dri",
+      "--socket=pulseaudio",
+      "--filesystem=host"
```

Comment:
Why is host needed?

Reply:
In PoS solutions there is a common practice of file exchange (one program writes something somewhere, and another reads it)
Because we just providing tools for file import/export/exchange we have no control where is actual "spooling directory" located in client machine